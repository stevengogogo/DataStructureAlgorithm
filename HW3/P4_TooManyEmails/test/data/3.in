3
OkayOkayOkay
x
Nooo
oo
JJJJJJ
j