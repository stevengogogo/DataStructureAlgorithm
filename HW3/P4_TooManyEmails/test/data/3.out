Okay|Okay|Okay
No
J|J|J|J|J|J
